doc/usecases.md

```diff
@@ -0,0 +1,57 @@
+#layer2
+
+Usage can refer to this [doc](layer2.md)
+
+## precondition
+
+The cluster should have at least two nodes, nodeA and nodeB.
+
+## test endpoint update
+1. create service test-svc which have one endpoint on nodeA.
```

Comment:
Please add some yaml here

Reply:
Just like this document: https://github.com/ahmetb/kubernetes-network-policy-recipes/blob/master/01-deny-all-traffic-to-an-application.md#example